README.md

```diff
@@ -34,7 +34,9 @@ Configure SparkPost to relay messages to
 
 ### Sendgrid
 
-Not implemented yet.
+Configure Sendgrid to relay messages to (with full email checkbox ticked)
```

Comment:
The documentation calls this "POST the raw, full email" -- that's the one, right?

https://sendgrid.com/docs/for-developers/parsing-email/setting-up-the-inbound-parse-webhook/

Reply:
that's the one!

Reply:
Altered the message